app/models/patient.rb

```diff
@@ -555,42 +559,36 @@ def send_assessment(force = false)
                   continuous_exposure ||
                   dependents_exclude_self.where('monitoring = ? OR continuous_exposure = ?', true, true).exists?
 
-    # If force is set, the preferred contact time will be ignored
-    unless force
+    # Determine if it is yet an appropriate time to send this person a message.
+    unless send_now
+      # Local "hour" (defaults to eastern if timezone cannot be determined)
       hour = Time.now.getlocal(address_timezone_offset).hour
+
       # These are the hours that we consider to be morning, afternoon and evening
       morning = (8..12)
       afternoon = (12..16)
       evening = (16..19)
-      if preferred_contact_time == 'Morning'
+
+      if preferred_contact_time&.downcase == 'morning'
         return unless morning.include? hour
-      elsif preferred_contact_time == 'Afternoon'
+      elsif preferred_contact_time&.downcase == 'afternoon'
         return unless afternoon.include? hour
-      elsif preferred_contact_time == 'Evening'
+      elsif preferred_contact_time&.downcase == 'evening'
         return unless evening.include? hour
+      else
+        # Default to roughly afternoon if preferred contact time is not specified
+        return unless (11..17).include? hour
       end
     end
 
-    # Default calling to afternoon if not specified
-    if (preferred_contact_method&.downcase == 'telephone call' ||
-        preferred_contact_method&.downcase == 'sms texted weblink' ||
-        preferred_contact_method&.downcase == 'sms text-message') && responder.id == id && preferred_contact_time.blank?
-      hour = Time.now.getlocal(address_timezone_offset).hour
-      return unless (11..17).include? hour
-    end
-
-    if preferred_contact_method&.downcase == 'sms text-message' && responder.id == id && ADMIN_OPTIONS['enable_sms'] && !Rails.env.test?
-      if !force
-        PatientMailer.assessment_sms(self).deliver_later
-      else
-        PatientMailer.assessment_sms_reminder(self).deliver_later
```

Comment:
Not sure why `assessment_sms_reminder` would ever be used. It appears to just send an SMS directly without going through the studio flow. Worth removing as well, or at very least renaming?

Reply:
It had a purpose at one point, but clearly not anymore. I'd rather eliminate that in a separate PR though, as I think this one already has a lot of scope creep.

Reply:
Agreed, and yeah that's fair. Made a very small ticket so we don't forget: https://tracker.codev.mitre.org/browse/SARAALERT-879